pkgs/development/python-modules/typer/default.nix

```diff
@@ -33,11 +33,10 @@ buildPythonPackage rec {
   propagatedBuildInputs = [
     click
     typing-extensions
-  ];
+  ] ++ optional-dependencies.standard;
```

Comment:
Since optional-dependencies should not be included in dependencies, I think it's better to leave a comment explicitly.